routers/router.go

```diff
@@ -3,9 +3,12 @@ package routers
 import (
 	"go-crawler-challenge/controllers"
 
-	beego "github.com/beego/beego/v2/server/web"
+	"github.com/beego/beego/v2/server/web"
 )
 
 func init() {
-	beego.Router("/", &controllers.MainController{})
+	web.Router("/", &controllers.MainController{})
```

Comment:
As we are using the default package name (`web`) instead of `beego` here, how about to use the default package name(`web`) everywhere instead of `beego`?

Reply:
Changed all `beego` to `web`